vcpkg-configuration.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Rather than having this file, you can simplify and use the built-in baseline on vcpkg.json, check what I had at https://github.com/Arcnor/Descent3/commit/b3b31ccd36ae0760b54f8574775236c57379cf51